sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -75,7 +75,7 @@ export class StudentService extends RecordDataModelService<Student> {
    * @returns true if the student was found, otherwise false.
    */
   async studentExists(studentId: number): Promise<boolean> {
```

Comment:
Great catch 😉

Reply:
If the `await` was added we are not returning a Promise anymore, right?